cmd/agent/install_mac_os.sh

```diff
@@ -202,10 +202,17 @@ cmd_agent="$cmd_real_user /opt/datadog-agent/bin/agent/agent"
 
 cmd_launchctl="$cmd_real_user launchctl"
 
-function new_config() {
+function sed_inplace_arg() {
     # Check for vanilla OS X sed or GNU sed
-    i_cmd="-i ''"
-    if [ "$(sed --version 2>/dev/null | grep -c "GNU")" -ne 0 ]; then i_cmd="-i"; fi
+    if [ "$(sed --version 2>/dev/null | grep -c "GNU")" -ne 0 ]; then
+        echo "-i"
+    fi
```

Comment:
```suggestion
    else
        echo "-i ''"
    fi
```

I think we're missing an `else` there, without it the `-i` is echoed twice:
```bash
sudo sh -c gsed -i -i '' 's/api_key...
```

Reply:
You're right :facepalm: thanks for the great catch, I'll fix this.